recipes/poppler/all/conanfile.py

```diff
@@ -132,125 +126,119 @@ def _minimum_compilers_version(self):
-        self._cmake.definitions["ENABLE_JPEG"] = self.options.with_libjpeg
-        self._cmake.definitions["WITH_PNG"] = self.options.with_png
-        self._cmake.definitions["WITH_TIFF"] = self.options.with_tiff
-        self._cmake.definitions["WITH_NSS3"] = self.options.with_nss
-        self._cmake.definitions["WITH_Cairo"] = self.options.with_cairo
-        self._cmake.definitions["ENABLE_GLIB"] = self.options.get_safe("with_glib", False)
-        self._cmake.definitions["ENABLE_GOBJECT_INTROSPECTION"] = self.options.get_safe("with_gobject_introspection", False)
-        self._cmake.definitions["WITH_GTK"] = self.options.get_safe("with_gtk", False)
-        self._cmake.definitions["WITH_Iconv"] = self.options.get_safe("with_libiconv")
-        self._cmake.definitions["ENABLE_ZLIB"] = self.options.with_zlib
-        self._cmake.definitions["ENABLE_LIBOPENJPEG"] = "openjpeg2" if self.options.with_openjpeg else "none"
-        if self.options.with_openjpeg:
-            # FIXME: openjpeg's cmake_find_package should provide these variables
-            self._cmake.definitions["OPENJPEG_MAJOR_VERSION"] = tools.Version(self.requires["openjpeg"].ref.version).major
-        self._cmake.definitions["ENABLE_CMS"] = "lcms2" if self.options.with_lcms else "none"
-        self._cmake.definitions["ENABLE_LIBCURL"] = self.options.with_libcurl
-
-        self._cmake.definitions["POPPLER_DATADIR"] = self.deps_user_info["poppler-data"].datadir.replace("\\", "/")
-        self._cmake.definitions["FONT_CONFIGURATION"] = self.options.fontconfiguration
-        self._cmake.definitions["BUILD_CPP_TESTS"] = False
-        self._cmake.definitions["ENABLE_GTK_DOC"] = False
-        self._cmake.definitions["ENABLE_QT5"] = self.options.with_qt and tools.Version(self.deps_cpp_info["qt"].version).major == "5"
-        self._cmake.definitions["ENABLE_QT6"] = self.options.with_qt and tools.Version(self.deps_cpp_info["qt"].version).major == "6"
-
-        self._cmake.definitions["ENABLE_CMS"] = "lcms2" if self.options.with_lcms else "none"
-        self._cmake.definitions["ENABLE_DCTDECODER"] = self._dct_decoder
-        self._cmake.definitions["USE_FLOAT"] = self.options.float
-        self._cmake.definitions["RUN_GPERF_IF_PRESENT"] = False
+            tc.variables["ENABLE_BOOST"] = self.options.splash
+
+        tc.variables["FONT_CONFIGURATION"] = self.options.fontconfiguration
+        tc.variables["WITH_JPEG"] = self.options.with_libjpeg
+        tc.variables["WITH_PNG"] = self.options.with_png
+        tc.variables["WITH_TIFF"] = self.options.with_tiff
+        tc.variables["WITH_NSS3"] = self.options.with_nss
+        tc.variables["WITH_Cairo"] = self.options.with_cairo
+        tc.variables["ENABLE_GLIB"] = self.options.get_safe("with_glib", False)
+        tc.variables["ENABLE_GOBJECT_INTROSPECTION"] = self.options.get_safe("with_gobject_introspection", False)
+        tc.variables["WITH_Iconv"] = self.options.get_safe("with_libiconv")
```

Comment:
Default false here too?

Reply:
What do you mean? For !with_cpp this should always be false (since the option doesn't exist) and otherwise it may be enabled or disabled 🤔

Reply:
Ah, now I understand, I don't need get_safe here :)